main.go

```diff
@@ -23,13 +24,23 @@ func init() {
 	flag.Parse()
 }
 
+func increment(num int) int {
+	return num + 1;
+}
+
 func main() {
 	readConfig(&sarpConfig)
 	checkConfig()
 
 	// Initialize Gin router
 	// gin.SetMode(gin.ReleaseMode)
 	r := gin.Default()
+
+	//Add Increment Function to Router
+	r.SetFuncMap(template.FuncMap{
+		"increment": increment,
```

Comment:
We could avoid having the function explicitly defined for everyone, and instead do something like:
```
	// Add... add function
	r.SetFuncMap(template.FuncMap{
		"increment": func(x int) int {
			return x + 1
		},
	})
```

Reply:
Makes sense. Got it!